data/blog/vasil-dimov-receives-lts-grant.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Long-Term Support For Vasil Dimov"
+date: '2023-10-16'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/10-vasil.jpg']
+summary: "OpenSats Welcomes Vasil Dimov as an LTS Grantee."
+---
+
+We are pleased to add Vasil Dimov to the list of OpenSats LTS grantees. Vasil is
+receiving long-term support for his ongoing contributions to Bitcoin Core.
+
+Vasil started contributing full-time to Bitcoin Core in 2020. Since then, he has
+worked on various improvements, focusing mostly on the networking layer of the
+software. In the last couple of years, he has worked on adding [Tor v3
+support](https://github.com/bitcoin/bitcoin/issues/18884), as well as adding
+support for [I2P](https://github.com/bitcoin/bitcoin/pull/20685) and
+[CJDNS](https://github.com/bitcoin/bitcoin/pull/23077). His [merged
+PRs](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+author%3Avasild+is%3Aclosed+sort%3Aupdated-desc)
+are a testament to his determination to improve P2P networking and privacy all
+around.
+
+As a seasoned and prolific developer, his contributions go beyond Bitcoin Core.
+His past work includes contributions to
+[c-lightning](https://github.com/ElementsProject/lightning/commits?author=vasild),
+as well as multiple bug fixes to various high-class projects such as git,
+OpenSSL, Xorg, wget, and others. You can find a full list of his contributions
+on his
+[website](https://people.freebsd.org/~vd/cv/_/vdcv.html#open_source_contribution_others).
+
+In addition to the various "mini projects" he is working on inside Core, he will
+continue to review pull requests. "I think it is equally important or even more
+important to do review rather than coding," he mentioned to us. "If a PR needs 4
+ACKs to get merged (on average) it means that I have to review 4 PRs for each PR
+I have opened myself."
```

Comment:
Not sure if somebody from the wider public may read that 4 literally (even though it says "on average").

```suggestion
important to do review rather than coding," he mentioned to us. "Every PR needs multiple
ACKs to get merged, so it means that I have to review multiple PRs for each PR
I have opened myself."
```